tests/common/plugins/conditional_mark/hooks/pre-commit.py

```diff
@@ -0,0 +1,25 @@
+import re
+import sys
+
+def main():
+    stage_files = sys.argv[1:]
+    for file in stage_files:
```

Comment:
The variable `file` would over shadow the built in `file` module. It would be better to use something like `stage_file`.

Reply:
done